x/interchainstaking/keeper/callbacks.go

```diff
@@ -273,7 +273,16 @@ func SigningInfoCallback(k *Keeper, ctx sdk.Context, args []byte, query icqtypes
 	}
 	if valSigningInfo.Tombstoned {
 		k.Logger(ctx).Error("Tombstoned validator found", "valoper", valSigningInfo.Address)
-		return fmt.Errorf("%q on chainID: %q was found to already have been tombstoned", valSigningInfo.Address, zone.ChainId)
+
+		valAddrBytes, err := addressutils.ValAddressFromBech32(valSigningInfo.Address, zone.GetValoperPrefix())
```

Comment:
Is this the consensus address, not the valoper?

Reply:
Yeah this is `consensus address`, I have a mistake here, and we have to make another `ICQ` to get the validator address from this `consensus address` 
cc: [here](https://github.com/cosmos/cosmos-sdk/blob/e65083d9e466d995772d4db6011f4a2c1e9b070b/x/staking/keeper/validator.go#L47)